desktop/python.ts

```diff
@@ -18,22 +19,19 @@ export async function ensurePython() {
   log.info('[ensurePython]', { message })
 }
 
-export async function ensureLibraries() {
+export async function ensureLibraries(props: { proxyUrls: types.IProxyUrls }) {
   log.info('[ensureLibraries]')
 
   const required = await readRequiredLibraries()
   const installed = await readInstalledLibraries()
   const missing = required.filter((spec) => !installed.includes(spec))
```

Comment:
why would there be libraries missing from the installer?

Reply:
It uses the concept of idempotency. The user at any point of time can have:
- no installed dependencies
- partially installed dependencies (if a new version of ODE adds a new dependency)
- partially outdated dependencies (if a new version of ODE bumps a dependency's version)

So the installer calculated the diff and apply `missing` parts

Reply:
@guergana I would also switch of course =)